.changeset/gentle-cats-dance.md

```
@@ -0,0 +1,30 @@
+---
+"task-master-ai": minor
+---
+
+Migrate AI services to use generateObject for structured data generation
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)